cloudify_marketplace/resources/replace_ssl_certificates

```diff
@@ -146,6 +148,19 @@ if __name__ == '__main__':
             AMQPINFLUX_CONFIG_PATH,
         ])
 
+        # Update the manager credentials
+        username = details['new_manager_username']
+        password = details['new_manager_password']
+        MANAGER_REST_SECUIRTY_FILE = '/opt/manager/rest-security.conf'
```

Comment:
For consistency, this should be at the top of the file with the other path definitions.

It should probably also have the correct spelling.